src/YieldDistributor.sol

```diff
@@ -97,8 +107,12 @@ contract YieldDistributor is IYieldDistributor, OwnableUpgradeable {
      * @return uint256 The voting power of the user
      */
     function getCurrentVotingPower(address _account) public view returns (uint256) {
-        return
-            this.getVotingPowerForPeriod(BREAD, lastClaimedBlockNumber - cycleLength, lastClaimedBlockNumber, _account);
+        return this.getVotingPowerForPeriod(
```

Comment:
I think it would be better to check if `BUTTERED_BREAD == address(0)` and just return `0` if so. This sort of creates a dependency loop because `ButteredBread.sol` needs to be initialized with the BREAD address and doesn't include a setter for it.